### DISCUSSION TITLE
We need a Discord server

### DISCUSSION THREAD

**Discussion by KaziMahbuburRahman** [saadeghi/daisyui#2670] [Ideas / feature requests]
I think DaisyUI should create a discord server 
So that we can discuss and solve small problems easily

**Comment by saadeghi** [comment]
It's better to use [GitHub discussions](https://github.com/saadeghi/daisyui/discussions) where it's searchable, indexable and can be integrated to issues and commits, instead of a closed, register-only chatroom app where discussions are not indexed on search engines and is not available to public.
